src/ui-test/tests/editor.actions.test.ts

```diff
@@ -57,12 +57,6 @@ describe('Camel file editor test', function () {
             const run = await editorView.getAction(CAMEL_RUN_ACTION_LABEL) as WebElement;
             await run.click();
 
-            await run.getDriver().wait(async function () {
-                console.log('Waiting for JBang dependencies...');
-                await bottomBar.openTerminalView(); // ensure terminal view is opened and focused
-                return (await new TerminalView().getText()).includes('[jbang] Dependencies resolved');
-            }, 60000);
```

Comment:
@djelinek Hi! 

Do we really need to check that dependencies were resolved (once), is it possible to get the next messages without dependencies resolution?

 (I have checked the try block from [here](https://github.com/camel-tooling/camel-dap-client-vscode/pull/386/files), but I do not understand why we need to wait if it was resolved once and if it is not possible to get Some "Routes startup" and/or "Hello Camel from yaml"  and/or etc without it)
 
 Can I delete it?

Reply:
you are right